index.html

```diff
@@ -206,5 +206,83 @@ <h4>Skills</h4>
         
     </section>
 </section>
+
+<footer>
+    <div class="images">
+        <div class="space">
+           <div class="image01"><img src="assets/Shape.png" alt="#" class="shape00"></div>
+           <div class="image02"><img src="assets/Shape_1.png" alt="#"  class="shape01"></div>
+           </div>
+        </div>
+        <div class=" box name"><h1>Get Started</h1></div>
```

Comment:
- [ ] Let's make sure our header tag follows Figma design in (font-weight, font-family, and font-size), let's also make sure we use h2 ~ h6 instead of `h1` since it's better semantically to have only one h1 per page 

Your Implementation | Figma Design
:----:|:----: 
![image](https://user-images.githubusercontent.com/87689547/149036086-e745a9b4-2af2-49ae-81cb-d4994888e45a.png) | ![image](https://user-images.githubusercontent.com/87689547/149036118-2efb9b29-2ba6-4cef-abf4-b3cb98d82d7d.png)